src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedThermoElasticWellbore/Example.rst

```diff
@@ -0,0 +1,256 @@
+  :language: xml
+  :start-after: <!-- SPHINX_TemperatureTimeHistory -->
+  :end-before: <!-- SPHINX_TemperatureTimeHistoryEnd -->
+
+Similarly, the following blocks are needed to collect the solid stress:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_SolidStressCollection -->
+  :end-before: <!-- SPHINX_SolidStressCollectionEnd -->
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_SolidStressTimeHistory -->
+  :end-before: <!-- SPHINX_SolidStressTimeHistoryEnd -->
+
+The displacement field can be collected for the whole domain using ``nodeManager`` as follows
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_DisplacementCollection -->
+  :end-before: <!-- SPHINX_DisplacementCollectionEnd -->
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_DisplacementTimeHistory -->
+  :end-before: <!-- SPHINX_DisplacementTimeHistoryEnd -->
+
+Also, periodic events are required to trigger the collection of this data on the mesh. For example, the periodic events for collecting the displacement field are defined as:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedThermoElasticWellbore_benchmark.xml
+  :language: xml
+  :start-after: <!-- SPHINX_DisplacementPeriodicEvent -->
+  :end-before: <!-- SPHINX_DisplacementPeriodicEventEnd -->
+
+
+---------------------------------
+Results and benchmark
+---------------------------------
+
+A good agreement between the GEOSX results and analytical results for temperature distribution around the cased wellbore is shown in the figures below:
+
+.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedThermoElasticWellbore/thermoElastic_casedWellbore_temperature.py
+
+.. _resultsCasedThermoElasticWellbore_temperatureFig:
+.. figure:: temperature.png
+   :align: center
+   :width: 500
+   :figclass: align-center
+
+   Temperature distribution around the cased wellbore
+
+and the validation for the radial displacement around the cased wellbore is shown below:
+
+.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedThermoElasticWellbore/thermoElastic_casedWellbore_displacement.py
+
+.. _resultsCasedThermoElasticWellbore_displacementFig:
+.. figure:: displacement.png
+   :align: center
+   :width: 500
+   :figclass: align-center
+
+   Radial displacement around the cased wellbore
+
+The validations of the total radial and hoop stress (tangent stress) components computed by GEOSX against reference results are shown in the figure below:
+
+.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedThermoElasticWellbore/thermoElastic_casedWellbore_stress.py
```

Comment:
GEOSX results (marker) are not shown in this figure, please double check it.

Reply:
done